src/main/scala/com/exasol/cloudetl/storage/StorageProperties.scala

```diff
@@ -123,10 +120,24 @@ object StorageProperties extends CommonProperties {
   def apply(params: Map[String, String], metadata: ExaMetadata): StorageProperties =
     new StorageProperties(params, Option(metadata))
 
-  /** Returns [[StorageProperties]] from only key-value pairs map. */
+  /**
+   * Returns [[StorageProperties]] from only key-value pairs map.
+   */
   def apply(params: Map[String, String]): StorageProperties =
     new StorageProperties(params, None)
 
+  /**
+   * Returns [[StorageProperties]] from [[java.util.Map]] key-value
+   * pairs map.
+   */
+  def apply(params: java.util.Map[String, String]): StorageProperties = {
+    val mmap = scala.collection.mutable.Map.empty[String, String]
```

Comment:
Please rename the variable. Is it a map with what?

Reply:
Fixed.